winter/schema/generation.py

```diff
@@ -27,8 +27,8 @@ def generate_swagger_for_operation(view_func, controller, controller_method: Con
     if output_serializer:
         responses[response_status] = output_serializer.class_(**output_serializer.kwargs)
     else:
-        responses[response_status] = openapi.Response(description='Success')
-    # TODO support response dataclasses, preferably implement in dataclasses.py to keep extensibility
+        type_info = inspect_type(controller_method.return_value_class)
```

Comment:
I think no
I think we shouldn't catch this error
Because it is better to register type
If you haven't register the type it is the error.
I think return something default is not so good

Reply:
It could be the case when for 100 endpoints schema is good, and for the only endpoint winter can't guess the schema. In the first place, we write code to serve HTTP API, not swagger.

Reply:
Added catching exception